website/docusaurus.config.js

```diff
@@ -9,6 +9,57 @@ const util = require("util");
 const webpack = require("webpack");
 const execAsync = util.promisify(require("child_process").exec);
 
+/**
+ * Modify the svgo configuration (in place) to prevent it from minifying IDs in SVGs
```

Comment:
This comment is missing a tiny bit more context on _why_ we don't want to minify IDs in SVGs